source/class/qx/ui/table/model/Abstract.js

```diff
@@ -292,6 +303,12 @@ qx.Class.define("qx.ui.table.model.Abstract", {
       }
 
       this.setColumnNamesByIndex(columnNameArr);
+    },
+
+    _checkEditing() {
+      if (this.getTable().isEditing()) {
+        throw new Error("A cell is currently being edited. Commit or cancel the edit before setting the table data");
```

Comment:
I just wonder if this is going to break anyone's app? I suspect there are cases of editing that would not have caused a failure. What do you think of adding a static `qx.ui.table.model.Abstract.THROW_ON_MODEL_CHANGE_DURING_EDIT` which defaults to `false` for now (and your app would set it to `true`), until qx v8 where we change the default to `true`, thereby maintaining backward compatibility and allowing an easy "fix" even in v8, for those with a valid and working use case?

Reply:
Yep, that's fine by me

Reply:
Is this a valid thing to do for a static?
```
  statics: {
    THROW_ON_MODEL_CHANGE_DURING_EDIT: qx.core.Environment.get("qx.version") >= "8.0.0"
  },
```

Reply:
I think so. It should probably have a comment, maybe even referencing the PR discussion about it, though.